examples/Benchmarking RDF against MDAnalysis.ipynb

```diff
@@ -32,8 +32,8 @@
    "outputs": [],
```

Comment:
<p>Change <code>r_maxs</code> to <code>r_maxes</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>